### DISCUSSION TITLE
Invalid format displayed when changing ID

### DISCUSSION THREAD

**Discussion by kidzgy** [rustdesk/rustdesk#9150] [General]
### Bug Description

The ID I changed meets the requirements and no matter what I enter it shows an invalid format.
![20240822220650](https://github.com/user-attachments/assets/13071e64-b0d0-4bd7-b330-0392df956be8)

### How to Reproduce

My system is Windows 11 23H2 and I use my own ID server .
After installing the program, the ID will be randomly generated, at this time, I entered my ID server address and KEY in the network, and after connecting successfully, I started to modify my ID name, and found that, no matter how I entered the name, it prompted an invalid format.

### Expected Behavior

I hope the name change works.

### Operating system(s) on local side and remote side

Win 11,ver 10.0.22631.3880

### RustDesk Version(s) on local side and remote side

1.3.0

### Screenshots

Picture above

**Comment by rustdesk** [comment]
Quite wierd, why below fails on your machine, might some underlying text encoding issue.

https://github.com/rustdesk/rustdesk/blob/5931af460e3efa01a4d42955837535a3b55ad325/libs/hbb_common/src/lib.rs#L238

**Comment by kidzgy** [comment]
Still not work in 1.3.1.

**Comment by lnlion** [comment]
Still not working in 1.3.2

**Comment by digitaloten** [comment]
<img width="580" height="331" alt="image" src="https://github.com/user-attachments/assets/a6f677e9-c389-496f-bb46-d3724e07e0e2" />

Running into the same issue on 1.4.6
Windows 11 25H2
